artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/PriorityLinkedListImpl.java

```diff
@@ -40,6 +41,10 @@ public class PriorityLinkedListImpl<E> implements PriorityLinkedList<E> {
 
    private int lastPriority = -1;
 
+   protected void removed(final int level, final E element) {
+      exclusiveIncrementSize(-1);
+   }
```

Comment:
Is this new method just for debugging? It doesnt seem to use either of the arguments, which seems odd given in one case new field tracking was added just to pass it.